packages/cli/src/credential.ts

```diff
@@ -39,12 +45,24 @@ credential
         message: 'Issuer DID',
       },
       {
-        type: 'list',
+        type: 'autocomplete',
         name: 'sub',
-        choices: identifiers.map((item) => ({
-          name: `${item.did} ${item.alias}`,
-          value: item.did,
-        })),
+        pageSize: 15,
+        source: async (answers: any, input: string) => {
+          const res = fuzzy
+            .filter(input, subjects)
+            .map((el: any) => (typeof el === 'string' ? el : el.original))
+          return res
+        },
+        validate: (val) => {
+          if (val && typeof val !== 'string') {
+            val = val.value
+          }
+          if (!val || !val.startsWith('did:')) {
```

Comment:
A subject can be a plain URL, it doesn't have to start with `did:`

It is even OK if the subject is missing, in which case the `credentialSubject.id` property of the credential should be omitted.

Reply:
if using anything other than a DID causes issues in other places, we can merge this as is and fix the other places later, but please flag it if that is the case